setup.py

```diff
@@ -30,7 +30,7 @@ def readfile(filename, split=False):
 
 setup(
     name="scaffoldfitter",
-    version="0.9.0",
+    version="0.10.0",
```

Comment:
It is better to change this just before a release is made. I will try and change this to a pyproject.toml file in another PR and make the version dynamic thus rendering manual manipulation redundant.

Reply:
No, just leave this as is.  I have further changes that will deal with this.